App/l10n/Resources/tr.lproj/BraveWallet.strings

```diff
@@ -334,6 +331,27 @@
 /* The title of the notification which will prompt at the top of the browser when users are visiting web3 site that is not yet connected with Brave Wallet. */
 "wallet.dappsConnectionNotificationTitle" = "Bu sayfa, Brave Cüzdanı'yla etkileşimde bulunmak istiyor";
 
+/* A message that will be displayed under the section header when there is no dapps have been granted wallet connection. */
+"wallet.dappsSettingsConnectedSitesSectionEmpty" = "Brave Cüzdanına bağladığınız DApps burada görüntülenir";
+
+/* The title of the section that will list all the permitted dapp connections */
+"wallet.dappsSettingsConnectedSitesSectionTitle" = "Bağlı DApps";
+
+/* The title of the section that will display settings for default wallet and dapp provider api access permission */
+"wallet.dappsSettingsGeneralSectionTitle" = "Genel";
+
+/* The navigation title of the settings screen for each Dapp. '%@' will be relpaced by the name of the coin type. For example, the title could 'Ethereum Settings', 'Solana Settings' etc. */
+"wallet.dappsSettingsNavTitle" = "%@ Ayarları";
+
+/* The warning message in a confirmation dialog when attempting to remove all wallet connection to a dapp. */
+"wallet.dappsSettingsRemoveAllWarning" = "Bunu yapmak %1$d %2$@ için tüm Cüzdan bağlantısı izinlerini kaldırır";
```

Comment:
No need for the second parameter here `%2$@`. Making website plural in Turkish will be a grammatical error.

Change it as `Bunu yapmak %1$d websitesi için tüm Cüzdan bağlantısı izinlerini kaldırır`